firstvoices/backend/management/commands/import_csv_data.py

```diff
@@ -108,6 +112,10 @@ def run_import():
         ("dict-altspellings", AlternateSpellingResource()),
         ("dict-pronunciations", PronunciationResource()),
         ("dict-categorylinks", DictionaryEntryCategoryResource()),
+        (
+            "base-characters-dict-entries-links",
```

Comment:
I think the name of the file here is going to cause an overlap issue. In line 104 all files starting with the string `base-characters` are going to be imported using the `CharacterResource`. You will need to change the name of your new task so the files have a unique starting string, maybe something like `character-dictionary-links`.